ssn/chapters/ModelPropertyInstance.html

```diff
@@ -1,17 +1,40 @@
 <h3>Generic or Specific Instances of sosa:Property</h3>
 <p>
-    The previous edition of SSN [[vocab-ssn-20171019]] left an ambiguity on whether an instance of <a
-        href="#SOSAProperty"><code>sosa:Property</code></a> should be generic to all features of interest (e.g.,
-    <code>ex:Temperature</code>, <code>ex:OnOffStatus</code>), or specific to a single feature of interest (e.g.,
-    <code>&lt;myBodyTemperature&gt;</code>, <code>&lt;LightStatus&gt;</code>).
-    This version solves the ambiguity by differentiating:
+    The SSN Ontology does not provide a general pattern for describing observable or actuatable properties.
+    It is recommended to use entries from one of the many existing catalogues of properties, such as the <a
```

Comment:
If you find it relevant: in most cases these properties vocabularies will be expressed with SKOS.